src/model/image-tag.ts

```diff
@@ -83,7 +83,7 @@ class ImageTag {
       case Platform.types.StandaloneWindows:
       case Platform.types.StandaloneWindows64:
         // Can only build windows-il2cpp on a windows based system
-        if (process.platform === 'win32') {
+        if (process.env.il2cppEnabled && process.platform === 'win32') {
```

Comment:
needs to be gathered into an input, the input can default to false for cloud runner

Reply:
@webbertakken from your view do you agree it makes sense to offer a new IL2Cpp input?

Currently, it seems we default to some expectations around using it, not sure if this actually forces IL2Cpp within the build, or just ensures needed dependencies are included?

Reply:
Was replaced with input, happy to discuss better options.